tools/runners/run-app.js

```diff
@@ -13,7 +13,7 @@ import { pluginVersionsFromStarManifest } from '../cordova/index.js';
 import { closeAllWatchers } from "../fs/safe-watcher";
 import { eachline } from "../utils/eachline";
 import { loadIsopackage } from '../tool-env/isopackets.js';
-import { once , EventEmitter, on }  from "events"
+
```

Comment:
Could you explain the reason for removing the Event Emmitter? 🤔

Reply:
I changed that code back to how it was for Meteor 2. Using promises and storing a single resolver function is easier to understand than how it was using event emitters - one reason is event emitters are much more flexible, so you have to be more careful when reading the code to make sure you understand how they are being used. Also, the terminology and api wasn't updated to match the implementation using event emitters, so we needed to either change the implementation or the api, and I chose to change the implementation.